ports/libmagic/portfile.cmake

```
@@ -31,9 +32,25 @@ if(VCPKG_TARGET_IS_WINDOWS)
     set(VCPKG_CXX_FLAGS "${VCPKG_CXX_FLAGS} -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_WARNINGS")
 endif()
 
+set(CONFIG_OPTIONS)
+
+foreach(feature zlib bzlib xzlib zstdlib)
```

```suggestion
foreach(feature IN ITEMS zlib bzlib xzlib zstdlib)
```

TBH I dislike most feature names. They match neither cababilities (preferred) nor port names (seen quite often).

---

Renamed features to match the ports they depend on.